pubspec.lock

```diff
@@ -519,7 +519,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.15"
+    version: "0.2.11"
```

Comment:
this is the decrease I am talking about

Reply:
I can approve the issue and merge it, just let me know about this so I can start learning this 🤓

Reply:
@isabelcosta  pubspec.lock is a autogenerated file, but if contributors are on different channels, we might face merge conflict fue to pubspec.lock file, I have a solution in mind I'll test it and comment back soon.
Thank You

Reply:
also @isabelcosta  i think we should keep the least approvals to merge to atleast 2 people

Reply:
That's probably the case @Techno-Disaster , I was using Flutter from `master`  and when I realized it was the bleeding-edge channel I immediately switched to `stable`. IMO we should only use `stable` channel.

Reply:
iirc I use the dev channel which is the latest, like alpha channel i think we should make ot mandatory for contributors to stick to one channel to avoid thos merge conflict cc @isabelcosta

Reply:
also @isabelcosta  the students PR thier contributions directly here during their GSoC  period or they PR their whole code after the GSoC period ends?

Reply:
yes this seems like a good idea, we should create `CONTRIBUTING.md` with such info.

Reply:
yep, will make it asap, thanks for bringing this out.
So are we sticking to the master channel or dev? @bartekpacia @isabelcosta 
master : stable
dev : bleeding edge, also these builds are pretty good to, altho they are called bleeding edge i have rarely found a bug in them

Reply:
I'd lean toward `stable`, especially that we don't have any tests currently.

Reply:
we will miss some features, but if everyone is in let's stick to stable (master) channel